ostd/src/lib.rs

```diff
@@ -48,6 +48,9 @@ pub mod timer;
 pub mod trap;
 pub mod user;
 
+#[cfg(feature = "coverage")]
+mod minicov;
```

Comment:
`minicov` is an implementation detail. Simply name this module `coverage` would be better.